src/components/AutoSelect.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <v-select v-bind="$attrs" v-on="$listeners" ref="vselect"></v-select>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import VSelect from "vuetify/lib/components/VSelect/VSelect";
+
+// AutoSelect is a wrapper around v-select that that autoselects the only
+// option. Bindings and listeners are forwarded to v-select.
+export default Vue.extend({
+  name: "AutoSelect",
+  inheritAttrs: false,
+  mounted() {
+    const vselect: VSelect = this.$refs.vselect;
+    this.autoselectOnlyOption(vselect);
+
+    this.$watch("$refs.vselect.allItems", () => {
+      this.autoselectOnlyOption(vselect);
+    });
+  },
+  methods: {
+    autoselectOnlyOption(vselect: VSelect) {
+      if (vselect.multiple) {
+        throw new Error("AutoSelect does not support multiple selections.");
+      }
+      const firstOption = vselect.allItems[0];
+      const isSelectionEmpty = vselect.selectedItems.length === 0;
```

Comment:
I simply don't know: This is always guaranteed to be an array even though you explicitly don't want this for multi selects?

Reply:
Exactly, v-select's internals are generalized like this to support both modes